src/transformState.js

```diff
@@ -4,8 +4,31 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
+      const obj = action.extraData;
```

Comment:
Not necessary to create this extra variable.
Use just `action.extraData` or destructure `action`